packages/client/src/containers/Login.container/SignupForm.js

```diff
@@ -52,14 +62,18 @@ const SignupForm = () => {
       {formErrors.password && (
         <p className="form-erros">{formErrors.password}</p>
       )}
-
       <button
         onClick={handleSubmit}
         className="signup-submit-button"
         type="submit"
       >
         Sign up
-      </button>
+      </button>{' '}
+      {Object.keys(formErrors).length === 0 && isSubmit ? (
+        <p className="success-message">Thank you for Signing up</p>
+      ) : (
+        <p>{message}</p>
```

Comment:
```suggestion
        <></>
```

Reply:
I get the error " there‘s no need for a Fragment at all" whenever I use <></> 
that is why I add an empty string.

Reply:
you can also use ('')

Reply:
Or just `null` if you don't want to render anything at all.